src/js/pyodide.ts

```diff
@@ -214,6 +222,18 @@ export async function loadPyodide(
     await loadScript(scriptSrc);
   }
 
+  let snapshot;
+  if (options._loadSnapshot) {
+    snapshot = await options._loadSnapshot;
+    if (snapshot?.constructor?.name === "ArrayBuffer") {
```

Comment:
Maybe the type for `_snapshot` needs to be updated?

Reply:
Changed to `Uint8Array | ArrayBuffer | PromiseLike<Uint8Array | ArrayBuffer>`.